lib/assets/javascripts/posix-bracket-expressions.js

```diff
@@ -4,445 +4,569 @@
-        + '\\u2cf2-\\u2cf3'
-        + '\\u2d00-\\u2d25'
-        + '\\u2d27'
-        + '\\u2d2d'
-        + '\\u2d30-\\u2d67'
-        + '\\u2d6f'
-        + '\\u2d80-\\u2d96'
-        + '\\u2da0-\\u2da6'
-        + '\\u2da8-\\u2dae'
-        + '\\u2db0-\\u2db6'
-        + '\\u2db8-\\u2dbe'
-        + '\\u2dc0-\\u2dc6'
-        + '\\u2dc8-\\u2dce'
-        + '\\u2dd0-\\u2dd6'
-        + '\\u2dd8-\\u2dde'
-        + '\\u2de0-\\u2dff'
-        + '\\u2e2f'
-        + '\\u3005-\\u3007'
-        + '\\u3021-\\u3029'
-        + '\\u3031-\\u3035'
-        + '\\u3038-\\u303c'
-        + '\\u3041-\\u3096'
-        + '\\u309d-\\u309f'
-        + '\\u30a1-\\u30fa'
-        + '\\u30fc-\\u30ff'
-        + '\\u3105-\\u312d'
-        + '\\u3131-\\u318e'
-        + '\\u31a0-\\u31ba'
-        + '\\u31f0-\\u31ff'
-        + '\\u3400-\\u4db5'
-        + '\\u4e00-\\u9fcc'
-        + '\\ua000-\\ua48c'
-        + '\\ua4d0-\\ua4fd'
-        + '\\ua500-\\ua60c'
-        + '\\ua610-\\ua62b'
-        + '\\ua640-\\ua66e'
-        + '\\ua674-\\ua67b'
-        + '\\ua67f-\\ua697'
-        + '\\ua69f-\\ua6ef'
-        + '\\ua717-\\ua71f'
-        + '\\ua722-\\ua788'
-        + '\\ua78b-\\ua78e'
-        + '\\ua790-\\ua793'
-        + '\\ua7a0-\\ua7aa'
-        + '\\ua7f8-\\ua801'
-        + '\\ua803-\\ua805'
-        + '\\ua807-\\ua80a'
-        + '\\ua80c-\\ua827'
-        + '\\ua840-\\ua873'
-        + '\\ua880-\\ua8c3'
-        + '\\ua8d0-\\ua8d9'
-        + '\\ua8f2-\\ua8f7'
-        + '\\ua8fb'
-        + '\\ua900-\\ua92a'
-        + '\\ua930-\\ua952'
-        + '\\ua960-\\ua97c'
-        + '\\ua980-\\ua9b2'
-        + '\\ua9b4-\\ua9bf'
-        + '\\ua9cf-\\ua9d9'
-        + '\\uaa00-\\uaa36'
-        + '\\uaa40-\\uaa4d'
-        + '\\uaa50-\\uaa59'
-        + '\\uaa60-\\uaa76'
-        + '\\uaa7a'
-        + '\\uaa80-\\uaabe'
-        + '\\uaac0'
-        + '\\uaac2'
-        + '\\uaadb-\\uaadd'
-        + '\\uaae0-\\uaaef'
-        + '\\uaaf2-\\uaaf5'
-        + '\\uab01-\\uab06'
-        + '\\uab09-\\uab0e'
-        + '\\uab11-\\uab16'
-        + '\\uab20-\\uab26'
-        + '\\uab28-\\uab2e'
-        + '\\uabc0-\\uabea'
-        + '\\uabf0-\\uabf9'
-        + '\\uac00-\\ud7a3'
-        + '\\ud7b0-\\ud7c6'
-        + '\\ud7cb-\\ud7fb'
-        + '\\uf900-\\ufa6d'
-        + '\\ufa70-\\ufad9'
-        + '\\ufb00-\\ufb06'
-        + '\\ufb13-\\ufb17'
-        + '\\ufb1d-\\ufb28'
-        + '\\ufb2a-\\ufb36'
-        + '\\ufb38-\\ufb3c'
-        + '\\ufb3e'
-        + '\\ufb40-\\ufb41'
-        + '\\ufb43-\\ufb44'
-        + '\\ufb46-\\ufbb1'
-        + '\\ufbd3-\\ufd3d'
-        + '\\ufd50-\\ufd8f'
-        + '\\ufd92-\\ufdc7'
-        + '\\ufdf0-\\ufdfb'
-        + '\\ufe70-\\ufe74'
-        + '\\ufe76-\\ufefc'
-        + '\\uff10-\\uff19'
-        + '\\uff21-\\uff3a'
-        + '\\uff41-\\uff5a'
-        + '\\uff66-\\uffbe'
-        + '\\uffc2-\\uffc7'
-        + '\\uffca-\\uffcf'
-        + '\\uffd2-\\uffd7'
-        + '\\uffda-\\uffdc'
```

Comment:
We should not use both in the hashtag regex, but there's no maintenance cost to keeping both available in the library.